### ISSUE TITLE
When using the Context Menu, I cannot configure the Sidebar to open automatically.

### ISSUE THREAD

**Issue by MintMain21** [ssborbis/ContextSearch-web-ext#471]
I decided to experiment with my use of the extension and try loading search results in the sidebar. A pop-up appeared asking me to manually open the sidebar. I searched through the extension's settings to see if I could set it to automatically open the sidebar, but the settings I've found that should toggle this ("Open the sidebar automatically when the page loads") does nothing, and I still have to manually open the sidebar. 

The Sidebar could actual help improve my use of what's already a great extension. I'm only reaching out to you about this cause I somehow suspect that this is a bug and not an intentional design limitation.

**Comment by ssborbis** [comment]
Some of the issue is there are two completely different usages of the word `sidebar`.

When I first wrote this addon, there was no Firefox Sidebar. At some point, early in development, I added an alternative to the quick menu and toolbar menu that could serve as an analogue to the Firefox search bar. I called it the Sidebar because it could be *pseuo-docked* to the side of the current web page. The majority of the settings you're seeing are for that.

Then Firefox released an API for what they called the Sidebar. It's basically a slightly nerfed web frame that clings to the side of the browser. Someone requested the option to open search results in the Firefox sidebar, so the Sidebar option was added to the Search Actions (new tab, background tab, etc ). 

Unfortunately, the method to open the Firefox sidebar available to developers requires the user to perform an action with the browser UI, such as clicking a context menu item or a toolbar button. It simply can't be opened from the majority of this addon's menus. It might be able to work with the context menu, but that's it. Otherwise, you'd need to open it yourself... every time.  It's a pain.
